eap74-rhel8-byos-multivm/src/main/bicep/modules/_uami/_uamiAndRoles.bicep

```diff
@@ -22,7 +22,7 @@ var name_deploymentScriptUserDefinedManagedIdentity = 'jboss-eap-vm-deployment-s
 var name_deploymentScriptContributorRoleAssignmentName = guid('${resourceGroup().id}${name_deploymentScriptUserDefinedManagedIdentity}Deployment Script')
 
 // UAMI for deployment script
-resource uamiForDeploymentScript 'Microsoft.ManagedIdentity/userAssignedIdentities@2022-01-31-preview' = {
+resource uamiForDeploymentScript 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31-preview' = {
```

Comment:
The version of userAssignedIdentities is not valid here, so please don't publish this to our offers. @edburns 
<img width="976" alt="image" src="https://github.com/Azure/rhel-jboss-templates/assets/4465723/0d7f5268-a7ca-42e2-9eb7-a9a98d0c0746">

Reply:
Thanks @backwind1233 . What version should it be?

Reply:
Thanks, I will change it to 2023-01-31.